packages/public-api/src/app.ts

```diff
@@ -1,20 +1,22 @@
 import express from 'express';
-import {
-  endpointMissingMiddleware,
-  errorHandlerMiddleware,
-  parseJson,
-  cors,
-} from './middleware/common';
-import healthCheckRouter from './healthcheck/router';
-import websiteRouter from './website/router';
+import * as trpcExpress from '@trpc/server/adapters/express';
+import { endpointMissingMiddleware, errorHandlerMiddleware, cors } from './middleware/common';
+import { appRouter, createContext } from './router';
 
 export function createApp() {
   const app = express();
 
-  app.use(cors);
-  app.use(parseJson);
-  app.use(healthCheckRouter);
-  app.use(websiteRouter);
+  if (process.env.NODE_ENV !== 'test') {
+    app.use(cors);
+  }
+
+  app.use(
+    '/',
+    trpcExpress.createExpressMiddleware({
```

Comment:
What's the purpose of the wildcard import here? Can't we just use `createExpressMiddleware` imported directly?